### PR TITLE
Menu Changes

### PR DESCRIPTION
**Pull request by mikemerryguy**
Adding 50 and 100mm moves to menu.

Adding Z probe Wizard to Motion menu vs so deep in the config- advanced-offsets menus

### Benefits

User interface

### Configurations

any config with any bed will show the motion (50,100mm)
any config andy board will show z probe wizard as long as it is enabled.

### Related Issues

other NEEDs for this 100 and 50 mm move would be translation I ONLY edited the english file.   If someone compiles with another language it will FAIL as the 50mm move was not in marlin at all prior to this.

### CODE REVIEW

File: Marlin/src/lcd/menu/menu_motion.cpp [hunk 1]
```diff
@@ -156,7 +156,8 @@ void _menu_move_distance(const AxisEnum axis, const screenFunc_t func, const int
     SUBMENU(MSG_MOVE_0001IN, []{ _goto_manual_move(IN_TO_MM(0.001f)); });
   }
   else {
-    if (LARGE_AREA_TEST) SUBMENU(MSG_MOVE_100MM, []{ _goto_manual_move(100); });
```
Comment (thisiskeithb) [inline Marlin/src/lcd/menu/menu_motion.cpp:159]: `LARGE_AREA_TEST` is defined here based on >=1,000mm bed sizes: https://github.com/MarlinFirmware/Marlin/blob/5809bdd400c77ef1e25b5215ef14d07671b9aaf6/Marlin/src/lcd/menu/menu_motion.cpp#L31

If the plan is to remove the 1,000mm bed size requirement, then more updates are needed:

https://github.com/MarlinFirmware/Marlin/blob/5809bdd400c77ef1e25b5215ef14d07671b9aaf6/Marlin/src/lcd/menu/menu_motion.cpp#L84

&

https://github.com/MarlinFirmware/Marlin/blob/5809bdd400c77ef1e25b5215ef14d07671b9aaf6/Marlin/src/lcd/menu/menu_motion.cpp#L153

Comment (mikemerryguy) [reply Marlin/src/lcd/menu/menu_motion.cpp:159]: Changes made and compiled on default examples. 
Updated all language files as well to add the 50mm move.
Didn't bother with the .5in as I really don't think many use anything other than metric.

Comment (thisiskeithb) [reply Marlin/src/lcd/menu/menu_motion.cpp:159]: > Didn't bother with the .5in as I really don't think many use anything other than metric.

We support inch mode (`INCH_MODE_SUPPORT`), so the change should be made for both.

Comment (mikemerryguy) [reply Marlin/src/lcd/menu/menu_motion.cpp:159]: Added the .5, not all language files even had inch mode defined.  So there were a lot less changes.  I am getting an error on checks on line 83 of edit on  menu_motion.cpp.  for some reason it doesn't show the error on my build, likely as it isn't used.

**Comment by mikemerryguy** [comment]
Checks failing due to having updated config files for my current printer to test.
successfully builds for LPC1769.  Will update printer and test within a day or two.

**Comment by thisiskeithb** [comment]
> Checks failing due to having updated config files for my current printer to test.

Your configs should not be pushed here. Please revert those changes.

**Comment by mikemerryguy** [comment]
Good point!  Back to default.

**Comment by mikemerryguy** [comment]
Also just thought about slim menu, might be a bit much for a user who has limited space, so the moves of 50 and 100 won't be added if on a slim menu  build.

**Comment by thisiskeithb** [comment]
> Also just thought about slim menu, might be a bit much for a user who has limited space, so the moves of 50 and 100 won't be added if on a slim menu build.

100mm moves don’t make sense on regular builds like common ~235mm for an Ender-3/clones (which is why this wasn’t made default for everyone).

Two turns of an encoder and you’re already near/at max axis length.

**Comment by mikemerryguy** [comment]
> > Also just thought about slim menu, might be a bit much for a user who has limited space, so the moves of 50 and 100 won't be added if on a slim menu build.
> 
> 100mm moves don’t make sense on regular builds like common ~235mm for an Ender-3/clones (which is why this wasn’t made default for everyone).
> 
> Two turns of an encoder and you’re already near/at max axis length.

Well 100mm moves maybe can be left in the large test area, but add 50
Large test area is extremely large though.
There are a lot of 300 and 400mm build areas with travels of 400 to 500mm.

Perhaps revamp and lower the large test area to 400mm?

Personally moving 100mm is nice on even a 235 bed.
Soft endstops keep it safe and during testing movement and calibrating printer it's nice to have one smooth planner move vs moving by 10mm and it shuddering as it completes each move.

Also perhaps keeping 100mm move on the Extruder would be a good idea as well since 100mm is the GO TO for calibrating esteps.

**Pull request merged**